dwds/lib/src/connections/debug_connection.dart

```
@@ -41,7 +52,16 @@ class DebugConnection {
 
   Future<void> close() =>
       _closed ??= () async {
-        await _appDebugServices.chromeProxyService.remoteDebugger.close();
+        // Close Chrome remote debugger if available
+        try {
+          final chromeProxyService = _appDebugServices.chromeProxyService;
+          if (chromeProxyService != null) {
+            await chromeProxyService.remoteDebugger.close();
+          }
+        } catch (_) {
+          // Chrome proxy service not available in WebSocket-only mode - ignore
```

Just like in the setup, it feels bad to see errors being swallowed like this. Can we catch the error we know we want to ignore and surface any others?